Adding oraclecloudapps.com from Oracle Autonomous Database

pmandala opened the pull request:
Public Suffix List (PSL) Pull Request (PR) Template
====

<!-- #### READ THIS FIRST ####

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

If you haven't yet, please read our guidelines:
https://github.com/publicsuffix/list/wiki/Guidelines#submit-the-change

Also, read them again, as many skip that part and 
get confused about why their PR is delayed or does
not get accepted when theirs didn't follow them.

A recent PR using the current template is 
https://github.com/publicsuffix/list/pull/1591, although 
the organization and description were not as substantial 
as desired, which required maintainers time to visit the 
requestors website to further research. 
Having more robust org/desc improves the PR processing 
pace due to the extra cycles not lost to research.
For an example of what an excellent description in a PR looks like
see https://github.com/publicsuffix/list/pull/615, 
although that example uses an earlier template.
-->
### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)
<!--
* [ ] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section
-->
__Submitter affirms the following:__ 
<!--
Third-party Limits are used elsewhere, such as at Cloudflare, Let's 
Encrypt, Apple, GitLab or others, and having an entry in the PSL alters 
the manner in which those third-party systems or products treat 
a given domain name or sub-domains within it.

To be clear, it is appropriate to address how those limits impact 
your domain(s) directly with that third-party, and it is inappropriate 
to submit entries to the PSL as a means to work around those limits or 
restrictions.

  * [ ] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)
    - [Cloudflare](https://developers.cloudflare.com/learning-paths/get-started/add-domain-to-cf/add-site/)
    - [Letsencrypt](https://letsencrypt.org/docs/rate-limits/)
    - <UPDATE THIS LIST WITH YOUR LIMITATIONS! REMOVE ENTRIES WHICH DO NOT APPLY! REMOVE THIS LINE!>
-->
<!--
The purpose of the question above is to expose limit workarounds.
If there are third party limits that the PR seeks to overcome, those
must be listed within the rationale section of this request, and 
provide a good level of detail the effort that was made to work directly 
with the third part(y|ies) in attempting to address this within their 
rationale response below.
In all cases, software and services should be discouraged from use of
the PSL as a rate-limiting tool, and provide clear instructions to their
own clients, partners and users on the manner in which they can directly
request rate limit increases.
We treat the following as an attestation in the public record of the 
requesting party that they are not attempting to bypass rate limits through
the PR.
-->

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

<!--
Submitter will maintain domains in good standing or may lose section.

The ongoing trust of the PSL requires it to be free of outdated or problematic entries. In making this pull request, there is a commitment by the submitter that they are going to review and maintain their relevant section. By submitting an entry, the requestor acknowledges that their entry and section may be removed if the domain does not maintain the respective _PSL entries in DNS, any domain(s) within their section fail to resolve in DNS, the domain does not get renewed, expires or is otherwise unreachable. Submitter further identifies that it is their responsibility to review their submitted section within the PSL, submitting updates or removals as their domain(s) may change over time. It is also the responsibility of the submitter to provide (and keep up to date) a reachable email address within the section, and to maintain that address as it may change over time, so that they receive notices.
-->

  * [x] The submitter acknowledges that it is their responsibility to maintain the domains within their section. This includes removing names which are no longer used, retaining the _psl DNS entry, responding to e-mails to the supplied address. Failure to maintain entries may result in removal of individual entries or the entire section.

<!--
The guidelines describe which section to place the entry, what the 
order of commented org placement, order of sorting of entries. 
(hint: TLD then SLD, Ascending sort)   Although it seems pedantic, 
the sorting and formatting rules help ensure all of the automation 
that uses the PSL operates correctly.  Typically both are solved or
neither.
-->

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

<!-- 
Sorting and formatting of the entries is outlined in the guidelines 
and non-conforming requests are one of the largest sources of delay,
so getting this right initially will aid successfully having it 
proceed.  Miss-located entries and trailing spaces should be avoided.
-->

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

<!-- 
Seriously, carefully read the downline flow of the PSL and the 
guidelines. Your request could very likely alter the cookie and 
certificate (as well as other) behaviours on your core domain name in 
ways that could be problematic for your business.

Rollback is really not predictable, as those who use or incorporate 
the PSL do what they do, and when. It is not within the PSL volunteers' 
control to do anything about that.  

The volunteers are busy with new requests, and rollbacks are lowest 
priority, so if something gets broken by your PR, it will potentially 
stay that way for an indefinite period of time (typically long).
-->

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---


<!--

As you complete each item in the checklist please mark it with an X

Example:

* [x] Description of Organization

-->

Description of Organization
====

Oracle Corporation is an American multinational computer technology company headquartered in Austin, Texas. In 2020, Oracle was the third-largest software company in the world by revenue and market capitalization.

Organization Website: https://www.oracle.com/
https://www.oracle.com/autonomous-database/

Reason for PSL Inclusion
====

<!--
Please tell us why your domain(s) should be listed in the PSL
(i.e. Cookie Security, Let's Encrypt issuance, IOS/Facebook, 
Cloudflare etc) and clearly confirm that any private section 
names hold registration term longer than 2 years and shall 
maintain more than 1 year term in order to remain listed.

If you are attempting to work around third party limits, use 
this area to describe how and detail the manner in which you 
have first attempted to engage those third parties on the 
matter.

Please also include the numbers of any past Issue # or PR # 
specifically related to this submission or section.

Three or more sentences here that describe the purpose for 
which your PR should be included in the PSL.  There is no 
upper limit, but six paragraphs seems like a rational stop.
-->
Oraclecloudapps,com, oraclegovcloudapps.com and oraclegovcloudapps.uk are for customer use when using the Oracle Cloud Infrastructure platform for Automonous Database. These domains are used for customers accessing oracle application deployed in against their database. 

Customer are reported Cookie Tossing issues hence we are fixing to deliver Customer web content must only be accessible via domains in the Public DNS Suffix List. 

Number of users this request is being made to serve: In millions

<!--
Identify if this is current or an estimate.
-->


DNS Verification via dig
=======

```
dig +short TXT _psl.oraclecloudapps.com
"https://github.com/publicsuffix/list/pull/2109"
```

```
 dig +short TXT _psl.oraclegovcloudapps.com
"https://github.com/publicsuffix/list/pull/2109"
```

```
dig +short TXT _psl.oraclegovcloudapps.uk 
"https://github.com/publicsuffix/list/pull/2109"
```

Results of Syntax Checker (`make test`)
=========

<!--
Please verify that you followed the correct syntax and nothing broke

git clone https://github.com/publicsuffix/list.git
cd list
make test

Simply let us know that you ran the test and those results
-->

```
========================================================================
Testsuite summary for libpsl 0.21.5
========================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
========================================================================
```

wdhdev commented:
* [ ]  Expiration (Note: Must STAY >2y at all times)     
  * [ ] oraclecloudapps.com expires 2025-06-17
  * [ ] oraclegovcloudapps.com expires 2025-08-08
  * [ ] oraclegovcloudapps.uk expires 2024-10-09
* [ ]  DNS _psl entries (Note: Must STAY in place)
  * [ ] `_psl.oraclecloudapps.com`
  * [ ] `_psl.oraclegovcloudapps.com`
  * [ ] `_psl.oraclegovcloudapps.uk`
* [x]  Tests pass
* [x]  Sorting
* [ ]  Reasoning/Organization description
* [x]  Non-personal email address

All domains in this PR are not renewed for >2y, also all of the `_psl` TXT records are missing.

You also state you are looking to get around Cloudflare and Letsencrypt limitations, can you please state why this is. Also you do not actually provide the reason as to why these domains should be included in the PSL, you only provide what these domains are used for. Please state reasons for the inclusion, e.g. cookie separation, URL highlighting, etc.

pmandala commented:
1. Working on the extending expiration of domain- Need some time here.
2. We are unable to make dig work. Could you please help how to debug on that?
3. Updated Reasoning in the main PR template 
4. There is a typo regarding Cloudflare limitation. We are not looking around it now. Fixed the template. Sorry for the confusion

wdhdev commented:
> Working on the extending expiration of domain- Need some time here.

That's alright, please note this won't be approved until then.

> We are unable to make dig work. Could you please help how to debug on that?

Not to be rude, but do you how DNS works? You need to place a `_psl` TXT record on each domain with this content: `https://github.com/publicsuffix/list/pull/2109`

This is used to verify that you control the domains you are requesting. You must work for Oracle in order to have this approved.

pmandala commented:
Fixed DNS records

wdhdev commented:
Can confirm they are in place. We just need the domain names renewed for >2y then we can likely get this approved. 😊

pmandala commented:
We fixed the expiration date of all domains.
oraclecloudapps.com expires 2033-08-07
 oraclegovcloudapps.com expires 2033-08-07
 oraclegovcloudapps.uk expires 2033-10-08

wdhdev commented:
@simon-friedberger This PR is ready to be merged. I confirmed the domain expiry dates via a WHOIS lookup.

dnsguru commented:
Running unit tests

dnsguru commented:
Have we confirmed submitter necus with Oracle?

dnsguru commented:
Have we confirmed submitter necus with Oracle?


Why is this separate from other Dyn entries?

pmandala commented:
[Oracle Cloud Autonomous Database Service ](https://www.oracle.com/autonomous-database/) serves few web applications which uses those domain names to access over public internet

groundcat commented:
[patch](https://patch-diff.githubusercontent.com/raw/publicsuffix/list/pull/2109.patch) of this PR shows:

```
From 9eb019968ee708e099618bbcf69a9a4821630bba Mon Sep 17 00:00:00 2001
From: Phaneendhar Mandala <phaneendhar.mandala@oracle.com>
Date: Tue, 20 Aug 2024 10:51:02 -0700
Subject: [PATCH] Adding oraclecloudapps.com from Oracle Autonomous Database
```

Observed the use of `oraclecloudapps.com` at https://docs.oracle.com/en/cloud/paas/autonomous-database/serverless/adbsb/apex-vanity-url.html

Observed the use of `oraclegovcloudapps.com` and `oraclegovcloudapps.uk` at https://blogs.oracle.com/machinelearning/post/announcing-new-urls-for-oracle-machine-learning-notebooks-and-restful-services

_psl TXT records are in place.

dnsguru commented:
That seems reasonable as "connective tissue" for evidence.  We on
confirming submitter.

We likely want to get these integrated with dyn entries vs a new section
like this

On Mon, Aug 26, 2024, 8:14 AM groundcat ***@***.***> wrote:

> patch
> <https://patch-diff.githubusercontent.com/raw/publicsuffix/list/pull/2109.patch>
> of this PR shows:
>
> From 9eb019968ee708e099618bbcf69a9a4821630bba Mon Sep 17 00:00:00 2001
> From: Phaneendhar Mandala ***@***.***>
> Date: Tue, 20 Aug 2024 10:51:02 -0700
> Subject: [PATCH] Adding oraclecloudapps.com from Oracle Autonomous Database
>
> Observed the use of oraclecloudapps.com at
> https://docs.oracle.com/en/cloud/paas/autonomous-database/serverless/adbsb/apex-vanity-url.html
>
> Observed the use of oraclegovcloudapps.com and oraclegovcloudapps.uk at
> https://blogs.oracle.com/machinelearning/post/announcing-new-urls-for-oracle-machine-learning-notebooks-and-restful-services
>
> _psl TXT records are in place.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/publicsuffix/list/pull/2109#issuecomment-2310459805>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACQTJJ43OBMGO2K76KTOIDZTNA7HAVCNFSM6AAAAABM2R54XSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGMJQGQ2TSOBQGU>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

simon-friedberger commented:
@pmandala Can you please move these to the section labeled `// Customer OCI - Oracle Dyn https://cloud.oracle.com/home https://dyn.com/dns/` as requested by @dnsguru ?

simon-friedberger commented on public_suffix_list.dat:
```diff
@@ -12158,6 +12158,12 @@ translated.page
 // Submitted by Jacob Lee <jacob@autocode.com>
 autocode.dev
 
+// Autonomous Database OCI - Oracle Dyn https://cloud.oracle.com/home https://dyn.com/dns/
```
Should be added here
 https://github.com/publicsuffix/list/pull/2109/files#diff-b4fc013fa0dff30019626bef00fba7a39210a002b463194b258b8b8ae302215bL12527

pmandala replied:
Resolved the order recommendation and moved after Customer-OCI

pmandala commented:
Rebased the the branch after new check in.

Submitting the tests again
 ```============================================================================
Testsuite summary for libpsl 0.21.5
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================

pmandala commented:
How do I reopen the this pull request?

pmandala commented:
Reopen the pull request after rebase

simon-friedberger commented:
You deleted the branch that it was trying to get added. Maybe you can reopen if you recreate that branch.

pmandala commented:
I already recreated that branch with the [same name adbs](https://github.com/pmandala/list/tree/adbs), still the `reopen and commit` is disabled

pmandala commented:
Reopen the branch

pmandala commented:
Reopen the branch

pmandala commented:
Reopen

pmandala commented:
reopen

pmandala commented:
reopen

pmandala commented:
Could some one please help in reopening this PR?

pmandala commented:
reopen

wdhdev commented:
Just create a new commit and open a new PR referencing this one.

pmandala commented:
New PR created https://github.com/publicsuffix/list/pull/2130
